pkg/bundle/client.go

```diff
@@ -155,6 +160,41 @@ func (bc *bundleClient) CreateClusterNamespace(ctx context.Context, clusterName
 	return nil
 }
 
+func (bc *bundleClient) CreateClusterConfigMap(ctx context.Context, clusterName string) error {
+	name := auth.ConfigMapName
+	namespace := api.PackageNamespace + "-" + clusterName
+	key := types.NamespacedName{
+		Name:      name,
+		Namespace: namespace,
+	}
+
+	cm := &v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+	}
+	err := bc.Get(ctx, key, cm)
+	if err == nil {
```

Comment:
Also in go, we always return from an error scenario and let the program flow unless an error is encountered.

Reply:
See previous comment explaining